### DISCUSSION TITLE
Indexing XML files withinthe MS Office file container

### DISCUSSION THREAD

**Discussion by chlcosta** [sepinf-inc/IPED#1967] [Q&A]
Hi,

Is there a setting to allow indexing XML files within the MS Office file container? In my case, I need to perform searches within the settings.xml files of MS Office files (docx and xlsx).

tks.

**Comment by lfcnassif** [comment]
Sorry for the delay. You can try to edit `conf/ParserConfig.xml` file and disable `OOXMLParser` so docx, xlsx and pptx files will be handled by `PackageParser`, expanding them as ordinary zip files. Don't forget to enable Documents, Spreadsheets and Presentations expansion in `conf/CategoriesToExpand.txt`.

But you may get a worse text indexing of those documents. Another approach would be creating a `MultipleParser` combining `OOXMLParser` and `PackageParser` to run both on OOXML files, there are some examples into ParserConfig.xml.
